taxi/tests/test_search.py

```diff
@@ -0,0 +1,75 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from taxi.models import Driver, Car, Manufacturer
+from django.urls import reverse
```

Comment:
Add a blank line between different import groups and order them correctly to follow PEP-8